staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -941,13 +948,31 @@ type PersistentVolumeClaimVolumeSource struct {
 	Hotpluggable bool `json:"hotpluggable,omitempty"`
 }
 
+// MemoryDumpVolumeSource represents a memory dump volume source.
+// Deprecated: Use ScratchVolumeSource with Type "memoryDump" instead. This type will be removed in a future version.
 type MemoryDumpVolumeSource struct {
 	// PersistentVolumeClaimVolumeSource represents a reference to a PersistentVolumeClaim in the same namespace.
 	// Directly attached to the virt launcher
 	// +optional
 	PersistentVolumeClaimVolumeSource `json:",inline"`
 }
 
+type ScratchVolumeType string
+
+const (
+	MemoryDumpType ScratchVolumeType = "memoryDump"
+)
+
+type ScratchVolumeSource struct {
```

Comment:
Not sure about using "ScratchVolume" here. CDI has the notion of scratch PVCs that are created/deleted by the cdi controller. People may think this is related. Maybe "utility" volume or something similar?